packages/framework/src/Model/Slider/SliderItem.php

```diff
@@ -72,6 +72,7 @@ public function __construct(SliderItemData $sliderItemData)
      */
     public function edit(SliderItemData $sliderItemData)
     {
+        $this->domainId = $sliderItemData->domainId;
         $this->name = $sliderItemData->name;
         $this->link = $sliderItemData->link;
         $this->hidden = $sliderItemData->hidden;
```

Comment:
I noticed (SonarCloud noticed actually) that the implementation of `::edit` method is the same as `__construct` is. 
Does it make sense to call the `edit` method from the constructor?

Reply:
It is "problem" a lot of entities and it is different "problem" in my opinion.

I don't know what is better, because in some case editation has little differences :-/

Reply:
Okay, resolving this conversation as wont-fix. Further refactoring should take place instead.